packages/roosterjs-editor-plugins/lib/plugins/Picker/PickerPlugin.ts

```diff
@@ -180,7 +182,12 @@ export default class PickerPlugin<T extends PickerDataProvider = PickerDataProvi
 
             case PluginEventType.KeyDown:
                 this.eventHandledOnKeyDown = false;
-                if (event.rawEvent.key == UNIDENTIFIED_KEY) {
+                // On Android Webview below 51, the key for KeyboardEvent is not supported and always returns undefined,
+                // so using the charCode property, which is 0 or 229.
+                const isLowVersionAndroid =
+                    event.rawEvent.key == undefined &&
+                    UNIDENTIFIED_CODE.indexOf(event.rawEvent.charCode) > -1;
+                if (event.rawEvent.key == UNIDENTIFIED_KEY || isLowVersionAndroid) {
```

Comment:
Since this check is used in two places should move it to a private method with a good name and comment?

Reply:
`isAndroidKeyboardEvent()` method is added to check Android keyboard events.